health-services/project-factory/src/server/utils/cryptUtils.ts

```
@@ -0,0 +1,45 @@
+import crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)